trafilatura/downloads.py

```diff
@@ -327,3 +326,12 @@ def _send_pycurl_request(url, no_ssl, config):
     # tidy up
     curl.close()
     return RawResponse(bufferbytes, respcode, effective_url)
+
+
+def _reset_global_objects():
```

Comment:
Is there a way to include the function directly in the tests instead of here?

Reply:
done in 178a44195af480294d62e4d3a6a3766cd9dc0929
Interestingly, mutating a global from another module is not so trivial. If you import it using `from module import X`: it doesn't work as it becomes part of the local module scope.
I had to use
```python
trafilatura.downloads.HTTP_POOL, trafilatura.downloads.NO_CERT_POOL, trafilatura.downloads.RETRY_STRATEGY = None, None, None
```